src/movie/movie.ts

```diff
@@ -273,6 +273,8 @@ export class Movie {
    * @param [options.type='video/webm'] - MIME type for exported video
    * @param [options.onStart] - Called when the recording starts
    * @return Resolves when done recording, rejects when media recorder errors
+   * @deprecated Use individual options instead of mediaRecorderOptions.
```

Comment:
It looks like `@deprecated` only applies to the entire function (https://stackoverflow.com/a/42457901/3783155). To only deprecate `mediaRecorderOptions`, you may need to add a docstring with `@deprecated` to `mediaRecorderOptions` in the `options` type definition below.